### PR TITLE
Eat cgstats

### PR DESCRIPTION
**Pull request by moonso**
## Description

This PR eats more of cgstats so that we can remove cgstats as a dependency from `requirements.txt`.
There where only two things that was needed:

- Move the declaration of the database models into CG
- Add find sample functionality into CG

The full functionality of cgstats is not yet in CG but now we can go on and step by step add the stuff that is still used in production.

### How to prepare for test

Tests are automated

## Review
- [x] code approved by MR
- [x] tests executed by GitHub
- [x] "Merge and deploy" approved by MM
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [x] **MINOR** - when you add functionality in a backwards compatible manner

### CODE REVIEW

**Comment by Mropat** [comment]
:hungary:

**Review by Mropat — APPROVED** [review]
I think you should eat the models

File: tests/meta/upload/gisaid/conftest.py [hunk 1]
```diff
@@ -81,16 +80,26 @@ def gisaid_case_id():
     return "gisaid_case"
 
 
+@pytest.fixture(name="gisaid_file_name")
```
Comment (Mropat) [inline tests/meta/upload/gisaid/conftest.py:83]: can this be saved to Path(tmpdir, "test.fasta")? then tmpdir is cleaned after pytest is done
even though this is rmoved once not used, still its kind of weird to save and delete file from repo base every time

**Review by Mropat — APPROVED** [review]

File: tests/meta/upload/gisaid/conftest.py (hunk 1 above)
Comment (moonso) [reply tests/meta/upload/gisaid/conftest.py:83]: Haha, yes that's exactly what I did first

Comment (moonso) [reply tests/meta/upload/gisaid/conftest.py:83]: But there is a test that checks for the filename in a file, so it is hardcoded there. And the tests will fail

**Comment by moonso** [comment]
Screen Shot 2021-04-27 at 09 22 50

deployed on hasta

**Pull request merged**